src/converse-resize-occupants-view.js

```diff
@@ -0,0 +1,161 @@
+     * false, which means these plugins are only loaded opportunistically.
+     *
+     * NB: These plugins need to have already been loaded via require.js.
+     */
+    dependencies: ["converse-muc-views", "converse-mouse-events"],
+
+    enabled (_converse) {
+        return _converse.allow_occupants_view_resizing;
+    },
+
+    overrides: {
+        // Overrides mentioned here will be picked up by converse.js's
+        // plugin architecture they will replace existing methods on the
+        // relevant objects or classes.
+        //
+        // New functions which don't exist yet can also be added.
+
+        ChatRoomView: {
+            afterShown () {
+                const { _converse } = this.__super__;
+
+                // set occupants width. Necessary if multiple MUCs are open.
+                const occupants_view = this.el.querySelector('.occupants');
+                occupants_view.style.flex = "0 0 " + _converse.occupantsview_width_pixel + "px";
+
+                this.__super__.afterShown.apply(this, arguments); 
+            }
+        },
+
+        ChatRoomOccupantsView: {
+            events: {
+                'mousedown .dragresize-roster-left': 'onStartHorizontalResizeRoster',
+            },
+
+            render () {
+                const { _converse } = this.__super__;
+                if(_.isNil(_converse.occupantsview_width_pixel)) {
+                    _converse.occupantsview_width_pixel = this.el.style.width;
+                }
+
+                const result = this.__super__.render.apply(this, arguments);
+                this.renderDragResizeHandles(this);
+                this.loadWidthOfOccupantsView();
+
+                this.is_maximum = false;
+                this.is_minimum = false;
+                
+                return result; 
+            },
+
+            renderDragResizeHandles (view) {
+                const flyout = view.el;
+                const div = document.createElement('div');
+                div.innerHTML = '<div class="dragresize dragresize-roster-left"></div>';
+                flyout.insertBefore(
+                    div,
+                    flyout.firstChild
+                );
+            },
+
+            loadWidthOfOccupantsView () {
+                const { _converse } = this.__super__;
+
+                if(!_.isNil(_converse.occupantsview_width_pixel)) {
+                    this.el.style.flex = "0 0 " + _converse.occupantsview_width_pixel + "px";
+                }
+            },
+
+            onStartHorizontalResizeRoster (ev) {
+                const flyout = this.el,
+                      style = window.getComputedStyle(flyout);
+                this.width = parseInt(style.width.replace(/px$/, ''), 10);
+                const { _converse } = this.__super__;
+                _converse.resizing = {
+                    'chatbox': this,
+                    'direction': 'left'
+                };
+                this.prev_pageX = ev.pageX;
+            },
```

Comment:
Methods such as this one, which don't exist on the original model being overridden, should rather be added via composition with `Object.assign` (you'll see lots of examples of this in other plugins).

Using `Object.assign` is a bit cleaner and explicit way of doing it, relying less on `pluggable.js` magic.